src/config/replicator.rst

```diff
@@ -23,15 +23,51 @@ Replicator Database Configuration
 
 .. config:section:: replicator :: Replicator Database Configuration
 
-    .. versionadded:: 1.2
+    .. config:option:: max_jobs
 
-    .. config:option:: max_replication_retry_count
+        .. versionadded:: 3.0
```

Comment:
Also here, might be 2.1 instead of 3.0, and for all such instances in this document. Pending discussion about which version should be the target in IRC/mailing lists :)